### ISSUE TITLE
Train IFNet_m

### ISSUE THREAD

**Issue by Eric-chuan** [hzwer/arXiv2021-RIFE#253]

**Comment by Eric-chuan** [comment]
Hi, I am trying to retrain IFNet_m and I observe that the loss_distill does not drop, here is my complete train_log, is this normal or how should I adjust it.
- single card
- batch_size = 64
- lr : 1e-5----1e-4
- loss = loss_l1 + loss_tea + loss_distill * 0.005
- 
![image](https://user-images.githubusercontent.com/45691282/165209958-4bc3f965-f8cc-460e-bcd2-096c37aa4a02.png)

**Comment by hzwer** [comment]
The psnr seem to be low. How about the visual affects?
Recently I release the dataset of RIFEm, did you check it? https://github.com/hzwer/arXiv2021-RIFE/blob/168b9d120ef65a25a166623a01b8fee4420da7e0/dataset.py#L59

**Comment by Eric-chuan** [comment]
Yes, I've synced your recent update on dataset.py, but the problem still exists. The following are the visual effects at t=0.25, t=0.5 and t=0.75 respectively.
![out](https://user-images.githubusercontent.com/45691282/165270130-ed9074e1-d77f-4d16-95f3-c0c1033c8663.png)

**Comment by hzwer** [comment]
There doesn't seem to be an essential error, but loss doesn't seem to match my record of #249.
I will start an experiment to check current code.

**Comment by hzwer** [comment]
You can add my WeChat hzwer1999.

**Comment by hzwer** [comment]
Our finally found that @Eric-chuan's train_log is normal. Because the teacher is changing during training, the distillation loss does not show a significant drop. The training of RIFEm may require better distillation methods.
